docs/concepts/mixer.md

```diff
@@ -7,40 +7,210 @@ title: Mixer
+- **Quota Management**. Enables services to allocate and free quota on a number of dimensions, Quotas are used as a relatively simple resource management 
 tool to provide some fairness between service consumers when contending for limited resources.
 
+## Attributes
+
+Istio uses *attributes* to control the runtime behavior of services running in the mesh. Attributes are named and typed pieces of metadata
+describing ingress and egress traffic and the environment this traffic occurs in. An Istio attribute carries a specific piece
+of information such as the error code of an API request, the latency of an API request, or the
+original IP address of a TCP connection. Here are a few examples of attributes:
+
+	request.path: xyz/abc
+	request.size: 234
+	request.time: 12:34:56.789 04/17/2017
+	source.ip: 192.168.0.1
+	target.service: example
+
+A given Istio deployment has a fixed vocabulary of attributes that it understands. The specific vocabulary is
+determined by the set of attribute producers being used in the deployment. The primary attribute producer in Istio
+is the proxy, although specialized mixer adapters and services can also introduce attributes. The common baseline
+set of attributes available in most mixer deployments is defined in *TBD*.
+
+## Request phases
+
+When a request comes in to the mixer, it goes through a number of distinct handling phases:
+
+- **Supplementary Attribute Production**. The first thing that happens in the mixer is to run a globally configured
+set of adapters (mixer plugins) that are responsible for introducing new attributes. These attributes are combined with the attributes
+from the request to form the total set of attributes for the operation.
+ 
+- **Resolution**. The second phase is to evaluate the set of attributes to determine the effective 
+configuration to apply for the request. See *TBD* for information on how resolution works. The effective
+configuration determines the set of aspects and descriptors available to handle the request in the
+subsequent phases.
+
+- **Attribute Processing**. The third phase takes the set of incoming request attributes
+and produces a set of policy objects. Attribute processing is initially
+configured through a simple declarative form as described in *TBD*. Advanced attribute processing can also
+be implemented through a general-purpose scripting environment as described [below](#scripting).
+
+- **Adapter Dispatching**. The Resolution phase establishes the set of available aspects and the Attribute
+Processing phase creates a set of policy objects. The Adapter Dispatching phase invokes the adapters
+associated with each aspect and passes them policy objects.
+
+<img style="display:block;width:40%;margin:auto;" src="./mixer/phases.svg" alt="Phases of mixer request processing" />
+<p style="text-align:center;">Request Phases</p>
+
+## Configuration state
```

Comment:
This section and the "Descriptors and policy objects" should come before Request Phases. They are similar to Attributes in that they describe what is being manipulated. After that, transition to the how seems more natural.